### PR TITLE
gui: Update connect statements to conform to Qt 5 standard

### PR DESCRIPTION
**Pull request by jamescowens**

### CODE REVIEW

**Comment by div72** [comment]
Maybe some changes in c2ed11e87ba68caf93fd6113a303f6ac241c225c could be helpful?

**Comment by jamescowens** [comment]
I had forgotten you did that...

**Comment by jamescowens** [comment]
You and I solved the arg squashing slightly differently. I created new wrapper functions that don't take an argument. You used lambdas...

connect(receiveCoinsPage, &AddressBookPage::signMessage, this, [this]{this->gotoSignMessageTab(QString());})

I am not sure which I like better...

**Comment by jamescowens** [comment]
Would you submit the rest in https://github.com/gridcoin-community/Gridcoin-Research/commit/c2ed11e87ba68caf93fd6113a303f6ac241c225c as a PR against my branch, and then it will propagate here? I would prefer one commit per major area changed.

**Comment by jamescowens** [comment]
@div72, I will go ahead and merge in the relevant parts of https://github.com/gridcoin-community/Gridcoin-Research/commit/c2ed11e87ba68caf93fd6113a303f6ac241c225c, since I think you are busy.

**Comment by div72** [comment]
@jamescowens I have some time now, if you haven't already did it.

**Comment by jamescowens** [comment]
Please go ahead! :)

**Comment by jamescowens** [comment]
The gear button to open the beacon configuration wizard does not seem to work. Accessing the wizard from the hamburger does. Will have to track that down.

File: src/qt/overviewpage.cpp [hunk 1]
```diff
@@ -306,11 +306,10 @@ void OverviewPage::setResearcherModel(ResearcherModel *researcherModel)
...
-    connect(researcherModel, SIGNAL(researcherChanged()), this, SLOT(updateResearcherStatus()));
-    connect(researcherModel, SIGNAL(magnitudeChanged()), this, SLOT(updateMagnitude()));
-    connect(researcherModel, SIGNAL(accrualChanged()), this, SLOT(updatePendingAccrual()));
-    connect(researcherModel, SIGNAL(beaconChanged()), this, SLOT(updateResearcherAlert()));
-    connect(ui->researcherConfigToolButton, SIGNAL(clicked()), this, SLOT(onBeaconButtonClicked()));
+    connect(researcherModel, &ResearcherModel::researcherChanged, this, &OverviewPage::updateResearcherStatus);
+    connect(researcherModel, &ResearcherModel::magnitudeChanged, this, &OverviewPage::updateMagnitude);
+    connect(researcherModel, &ResearcherModel::accrualChanged, this, &OverviewPage::updatePendingAccrual);
+    connect(researcherModel, &ResearcherModel::beaconChanged, this, &OverviewPage::updateResearcherAlert);
 }
```
Comment (div72) [inline src/qt/overviewpage.cpp:314]:
```suggestion
    connect(ui->researcherConfigToolButton, &QAbstractButton::clicked, this, &OverviewPage::onBeaconButtonClicked);
}
```

Comment (jamescowens) [reply src/qt/overviewpage.cpp:314]: Yep.

**Comment by jamescowens** [comment]
@div72 You want to review this one more time? I think we are good now.

**Review by div72 — APPROVED** [review]
Code Review ACK, seems like QT 5.15 moved to errorOccured for the error signal. Would you like to add that in with a conditional since we're already modifying there?

**Comment by jamescowens** [comment]
Ok. I think that does it.

**Comment by cyrossignol** [comment]
> You and I solved the arg squashing slightly differently. I created new wrapper functions that don't take an argument. You used lambdas...
> 
> connect(receiveCoinsPage, &AddressBookPage::signMessage, this, [this]{this->gotoSignMessageTab(QString());})
> 
> I am not sure which I like better...

One potential consideration: the lambdas have internal linkage which optimizes them out. The wrapper functions will likely remain in some binary form since they're part of the external API of the class.

**Comment by jamescowens** [comment]
Yeah. Your point has swayed me. I think I will convert those to lambdas. Almost done with this one!

**Comment by jamescowens** [comment]
@div72 please take one final look at this. I changed to the lambdas...

File: src/qt/bitcoingui.cpp [hunk 2]
```diff
@@ -1587,11 +1578,6 @@ void BitcoinGUI::showNormalIfMinimized(bool fToggleHidden)
         hide();
 }
 
-void BitcoinGUI::showNormalIfMinimizedDefault()
-{
-    showNormalIfMinimized(false);
-}
-
 void BitcoinGUI::toggleHidden()
```
Comment (div72) [inline src/qt/bitcoingui.cpp]: Could you also axe this?

Comment (jamescowens) [reply src/qt/bitcoingui.cpp]: Done.

**Pull request merged**